examples/replica_set_single_disk/values.yaml

```diff
@@ -83,3 +83,22 @@ additionalUsers:
         role: "readWriteAnyDatabase"
       - db: admin
         role: "userAdminAnyDatabase"
+        
+backup:
+  mode: enabled
+  autoTerminateOnDeletion: false
+  encryption:
+    kmip:
+      client:
+        clientCertificatePrefix: ""
+  snapshotSchedule:
```

Comment:
Just for the record, the same thing happens here. Providing empty values is not the same as not providing this at all.

Reply:
Also added validation to verify if value of clientCertificatePrefix is empty string, then to not include encryption block at all.